raiden/network/transport/matrix/transport.py

```diff
@@ -888,7 +894,13 @@ def _initialize_health_check(self, health_check_list: List[Address]) -> None:
         )
         gevent.joinall(greenlets, raise_error=True)
 
-    def _extract_addresses(self, room: Room) -> List[Address]:
+    """
```

Comment:
the docstring is floating in the class body, it should be the first line after the function definition ref: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
thanks